.github/workflows/scripts/detect-jobs-to-run.js

```diff
@@ -1,29 +1,11 @@
 #!/usr/bin/env node
 // @ts-check
-
-const { stdin } = process
+const streamConsumer = require('node:stream/consumers')
 const fs = require('fs')
 
-// From https://github.com/sindresorhus/get-stdin/blob/main/index.js
-async function getStdin() {
-  let result = ''
-
-  if (stdin.isTTY) {
-    return result
-  }
-
-  stdin.setEncoding('utf8')
-
-  for await (const chunk of stdin) {
-    result += chunk
-  }
-
-  return result
-}
```

Comment:
ℹ️  We got some `detectOpenHandles` issues due to the get-stdin package and hence switched to the more modern node native approach. Also changing this code that was copied out of that lib for consistency.